client.go

```diff
@@ -22,7 +22,7 @@ package cluster
 
 import (
 	"github.com/m3db/m3cluster/kv"
-	"github.com/m3db/m3x/watch"
+	xwatch "github.com/m3db/m3x/watch"
```

Comment:
that way the mockgen would fail, it could not find the xwatch package, only works with rename somehow.